### PR TITLE
Refactor deployment info

### PR DESCRIPTION
**Pull request by bgrozev**

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -106,5 +106,12 @@ var config = { // eslint-disable-line no-unused-vars
         // How long we're going to wait, before going back to P2P after
         // the 3rd participant has left the conference (to filter out page reload)
         //backToP2PDelay: 5
+    },
+    // Information about the jitsi-meet instance we are connecting to, including the
+    // user region as seen by the server.
+    deploymentInfo: {
```
Comment (bbaldino) [inline config.js]: so we define this here, but it's also in the infra pr...how does it get inserted? that one overrides this one?

File: modules/analytics/analytics.js [hunk 2]
```diff
@@ -122,15 +122,11 @@ class Analytics {
...
-                 * optionally include local deployment information,
-                 * window.jitsiDeploymentInfo defined outside of application
-                 * to use, override contents in local.html
-                **/
-                if (window.jitsiDeploymentInfo) {
-                    for (var key in window.jitsiDeploymentInfo) {
-                        permanentProperties[key] 
-                            = window.jitsiDeploymentInfo[key];
+                 // optionally include local deployment information based on
+                 // the contents of window.config.deploymentInfo
```
Comment (bbaldino) [inline modules/analytics/analytics.js]: is this comment still accurate (with regards to 'window')?

File: modules/analytics/analytics.js [hunk 3]
```diff
@@ -122,15 +122,11 @@ class Analytics {
...
+                if (config.deploymentInfo) {
+                    for (var key in config.deploymentInfo) {
```
Comment (bbaldino) [inline modules/analytics/analytics.js]: does this need the `hasOwnProperty` check inside?  also, can this be a `let`?

File: modules/analytics/analytics.js (hunk 2 above)
Comment (bgrozev) [inline modules/analytics/analytics.js]: Yeah, "config" is a global.

File: modules/analytics/analytics.js (hunk 3 above)
Comment (bgrozev) [inline modules/analytics/analytics.js]: Fixed

**Pull request merged**